packages/common/src/dbt/validation.ts

```diff
@@ -0,0 +1,158 @@
+import Ajv, { ValidateFunction } from 'ajv';
+import addFormats from 'ajv-formats';
+import { AnyValidateFunction } from 'ajv/dist/types';
+import {
+    DbtManifestVersion,
+    DbtMetric,
+    DbtModelNode,
+    DbtRawModelNode,
+} from '../types/dbt';
+import { ParseError, UnexpectedServerError } from '../types/errors';
+import assertUnreachable from '../utils/assertUnreachable';
+import lightdashMetadataSchema from './schemas/lightdashMetadata.json';
+import dbtManifestSchemaV8 from './schemas/manifestV8.json';
+import dbtManifestSchemaV9 from './schemas/manifestV9.json';
+
+const LightdashMetadataSchemaRef =
+    'https://schemas.lightdash.com/dbt/manifest.json';
+
+const getDbtManifestWithLightdashMetadata = (dbtSchemaRef: string) => ({
```

Comment:
this is ok for  v8 and v9 have the same definitions, but adding support (again) for v7 , or a potentially a different future version, might not look great. What if we create a separate "static" version instead ?

Reply:
yes, that makes sense 👍 
Will change it.